styles.css

```diff
@@ -1,32 +1,103 @@
-form {
-  width: 10%;
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Comic Neue', cursive;
+}
+
+.list {
   display: flex;
   flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  border: solid 3px black;
+  margin-top: 1rem;
+  padding: 1rem 0;
+  font-family: 'Comic Neue', cursive;
+}
+
+h1 {
+  display: flex;
   justify-content: center;
+  font-size: 40px;
+  font-family: 'Comic Neue', cursive;
+  font-weight: 700;
+}
+
+h2 {
+  font-size: 40px;
+  font-family: 'Comic Neue', cursive;
+  font-weight: 700;
+}
+
+.add_new {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] The requirement is that we follow the given wireframe with our design. Kindly fix your design so that it matches the given wireframe.

Given wireframe
![image](https://github.com/AndresEspi/AwesomeBooks/assets/81932478/7f6f8940-8fc9-4296-9763-6645bd6de8e0)

Your design
![image](https://github.com/AndresEspi/AwesomeBooks/assets/81932478/1b191548-6921-4277-b8cf-6895ece4346d)